src/Npgsql/NpgsqlBuffer.cs

```diff
@@ -618,5 +618,71 @@ struct BitConverterUnion
         }
 
         #endregion
+
+        #region Postgis
+        //  Postgis returns bytes in ndr representation.
+        internal int PostgisReadInt32()
```

Comment:
One more note: these methods will break on a big endian system, can you please fix that?